gensim/models/hdpmodel.py

```diff
@@ -507,9 +508,13 @@ def save_options(self):
             fout.write('eta: %s\n' % str(self.m_eta))
             fout.write('gamma: %s\n' % str(self.m_gamma))
 
-    def hdp_to_lda(self):
+    def hdp_to_lda(self, model=False):
```

Comment:
I would rather deprecate the older behaviour of returning `alpha` and `beta` and only have the feature of returning a model. There should also be parameters on number of iterations and gamma_threshold to be used for the LdaModel. See [this](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/wrappers/ldamallet.py#L354) for an example.